libafl/src/stages/concolic.rs

```diff
@@ -39,10 +39,10 @@ where
     E: UsesState<State = TE::State>,
     EM: UsesState<State = TE::State>,
     TE: Executor<EM, Z> + HasObservers,
-    TE::State: HasExecutions + HasCorpus,
+    TE::State: HasExecutions + HasCorpus + HasNamedMetadata,
     Z: UsesState<State = TE::State>,
 {
-    type Progress = (); // stage cannot be resumed
+    type Progress = (); // stage internally handles progress
```

Comment:
`type HandlesProgressInternally = ()`?

Reply:
Nice, I'll do this one for now but this is probably worthwhile to add elsewhere as well.